installers/msi-language/StateDeploy/CustomAction.cs

```diff
@@ -228,7 +229,7 @@ public static ActionResult InstallStateTool(Session session, out string stateToo
             {
                 string msg = string.Format("Could not update PATH. Attempted to set path to: {0}, encountered exception: {1}", newPath, e.ToString());
                 session.Log(msg);
-                ActiveState.RollbarHelper.Report(msg);
+                RollbarReport.Error(msg);
```

Comment:
This makes the install fail right? So should be critical

Reply:
Yes, you're right. I think we updated this to specifically fail if we can't update PATH